CMakeLists.txt

```diff
@@ -293,6 +266,32 @@ if (PARTHENON_ENABLE_HOST_COMM_BUFFERS)
   endif()
 endif()
 
+if (Kokkos_ENABLE_CUDA AND TEST_INTEL_OPTIMIZATION)
+  message(WARNING
+    "Intel optimizer flags may not be passed through NVCC wrapper correctly. "
+    "If you encounter problems, please delete your CMake cache "
+    "and rerun CMake with -DTEST_INTEL_OPTIMIZATION=OFF.")
```

Comment:
Is this an issue with the new LLVM-based Intel, or the old Intel compiler, or both?

Reply:
Good point.
If it's up to me I'd even remove these this and, if people actually use it, let them move it to a machine specific config.

Reply:
I agree that would be cleaner.

Reply:
Any strong feelings here by anyone?

Reply:
Not really.